copter/source/docs/aerobatic-multicopter.rst

```diff
@@ -0,0 +1,42 @@
+.. _aerobatic-multicopter:
+
+============================
+Aerobatic/Racing Multicopter
+============================
+
+**this page is a work in progress and is incomplete currently**
+
+With the proper vehicle configuration and careful tuning, a precise, aggressive multicopter can be obtained for racing or aerobatics in ACRO mode.
+
+The first requirement is a vehicle frame that is very stiff to resist resonant vibrations and a power system that delivers high thrust to weight ratios. This is most easily obtained with smaller (< 6inch diagonal motor distances) frames.
```

Comment:
6 inch diagonal motor distance is a pretty tiny frame (and actually a bit harder to get high thrust to weight ratios at that size).
Are you sure you don't mean 6 inch diameter props?
Although I would say 7 inch prop diameter quads are also pretty easy to get high thrust to weight ratios on.

Reply:
yeah should be 6inch diameter prop frames.... pretty much the upper limit on racing/aero miniquads....bigger makes for more drag in high speed aerobatics...most are 4-5inch these days in MultiGP

Reply:
Yeah, nearly all quad racing (other than tinywhoops) is done with 5 inch prop quads, but 7 inch quads can still move - my Braveheart quad is a 7 inch one. Got up to some pretty crazy speeds.